index-handlers/src/main/java/no/sikt/nva/nvi/index/utils/SearchConstants.java

```diff
@@ -40,100 +27,63 @@ public final class SearchConstants {
     public static final String ASSIGNEE = "assignee";
     public static final String NUMBER_OF_APPROVALS = "numberOfApprovals";
     public static final String APPROVALS = "approvals";
-    public static final String PUBLICATION_DETAILS = "publicationDetails";
     public static final String PUBLICATION_DATE = "publicationDate";
     public static final String YEAR = "year";
-    public static final String MONTH = "month";
-    public static final String DAY = "day";
     public static final String KEYWORD = "keyword";
     public static final String APPROVAL_STATUS = "approvalStatus";
     public static final String PUBLICATION_DETAILS = "publicationDetails";
     public static final String CONTRIBUTORS = "contributors";
     public static final String NAME = "name";
     public static final String AFFILIATIONS = "affiliations";
+    public static final String PART_OF = "partOf";
     public static final String ROLE = "role";
     public static final String NVI_CANDIDATES_INDEX = "nvi-candidates";
     public static final String SEARCH_INFRASTRUCTURE_CREDENTIALS = "SearchInfrastructureCredentials";
     public static final Environment ENVIRONMENT = new Environment();
     public static final String SEARCH_INFRASTRUCTURE_API_HOST = readSearchInfrastructureApiHost();
     public static final String SEARCH_INFRASTRUCTURE_AUTH_URI = readSearchInfrastructureAuthUri();
+    public static final String JSON_PATH_DELIMITER = ".";
```

Comment:
I'm tempted to say that we should remove the class in its entirety…but as a new task so you can get this one merged